pom.xml

```diff
@@ -10136,10 +10136,13 @@
                 <jdk>[9,)</jdk>
             </activation>
             <properties>
-                <modular.jdk.args>--add-exports=java.base/sun.nio.ch=ALL-UNNAMED
+                <default.modular.jdk.args>--add-exports=java.base/sun.nio.ch=ALL-UNNAMED
                     --add-exports=jdk.unsupported/sun.reflect=ALL-UNNAMED
                     --add-exports=jdk.unsupported/sun.misc=ALL-UNNAMED
-                    --add-modules=java.se</modular.jdk.args>
+                    --add-modules=java.se
```

Comment:
I identified few more needed parameters here: 
https://github.com/wildfly/wildfly/pull/14304/commits/b3706a5989381175b042977b4ee61092ca0b4858
If you will include them @bstansberry I can close my PR as duplicate of this one.

Reply:
@ropalka We're going to need to write down an explanation for all of these.

Reply:
I unfortunately forgot to write down the reason for java.util and java.security. https://github.com/wildfly/wildfly-core/pull/4591 has some code comment notes about java.naming and javax.management (although some better place  to record this would be good.)